packages/EasyErrorHandler/src/Bridge/Laravel/Handler/Handler.php

```diff
@@ -190,7 +190,7 @@ protected function getResponseStatusCode(Exception $exception): int
      */
     protected function isExtendedResponse(): bool
     {
-        return $this->config->get('easy-error-handler.use_extended_response', false) === true;
+        return (bool)$this->config->get('easy-error-handler.use_extended_response', false) === true;
```

Comment:
I'm always confuse with casting, does it apply only to the left side of the expression?

If not, we might have some surprises. Maybe we should be cautious and write it this way:

```php
$responseExtended = (bool)$this->config->get('easy-error-handler.use_extended_response', false);

return $responseExtended === true;
```

Reply:
Done.